src/core/meshcop/tcat_agent.hpp

```diff
@@ -256,6 +256,19 @@ class TcatAgent : public InstanceLocator, private NonCopyable
         kStateConnected,
     };
 
+    /**
+     * Represents Device ID type.
+     *
+     */
+    enum TcatDeviceIdType : uint8_t
+    {
+        kTcatDeviceIdEmpty         = OT_TCAT_DEVICE_ID_EMPTY,         ///< Vendor device ID type not set
```

Comment:
the comments here duplicate the ones for the defined values made in tcat.h.  Maybe they could be removed here?

Reply:
fixed.